Makefile

```diff
@@ -44,7 +44,12 @@ build-requirements:
 update-kinto-admin:
 	rm -fr kinto_admin/static/*
 	npm install -g kinto-admin
-	kinto-admin build -d kinto_admin/static/
+	rm -fr tmp
+	git clone https://github.com/Kinto/kinto-admin.git tmp
+	cd tmp; git checkout v$$(kinto-admin --version); npm install
```

Comment:
I'm confused as to why we still need the kinto-admin package installed from npm? Now we're cloning the repo, we could simply checkout the tag and build from there. That would also allow specifying the version explicitly which would prevent possible *surprises*.